web/webhook_test.go

```diff
@@ -294,7 +294,7 @@ func TestCommandWebhooks(t *testing.T) {
 
 	for i := 0; i < 5; i++ {
 		response, appErr2 := http.Post(ApiClient.Url+"/hooks/commands/"+hook.Id, "application/json", bytes.NewBufferString(`{"text":"this is a test"}`))
```

Comment:
This one shouldn't be called `appErr2` as it's an `error`.

Reply:
Done :)